scripts/wmt14/README

```diff
@@ -0,0 +1,20 @@
+For machine translation (MT) tasks, all examples should be in the format
+
```

Comment:
Make sure you indicate in the jiant.info documentation that the documentation for MT is in scripts/wmt14/README.md